### PR TITLE
fix casing for product names

### PR DESCRIPTION
**Pull request by dimitropoulos**
### Description

I happened to notice, while reading the docs, a few typo'd product names.  This PR makes them correct and consistent.

### Testing Instructions

Should not be any behavioral changes.

### CODE REVIEW

File: docs/repo-docs/guides/tools/typescript.mdx [hunk 1]
```diff
@@ -73,7 +73,9 @@ Inside `packages/typescript-config`, you have a few `json` files which represent
 }
 ```
 
-<LinkToDocumentation href="https://www.typescriptlang.org/tsconfig">`tsconfig` options reference</LinkToDocumentation>
+<LinkToDocumentation href="https://www.typescriptlang.org/tsconfig">
+  `tsconfig` options reference
```
Comment (anthonyshew) [inline docs/repo-docs/guides/tools/typescript.mdx:77]: This newline will mess up the rendered output by adding an extra <p> tag.

**Review by anthonyshew — COMMENTED** [review]
Looks good other than the newline thing.

File: docs/repo-docs/guides/tools/typescript.mdx (hunk 1 above)
Comment (dimitropoulos) [reply docs/repo-docs/guides/tools/typescript.mdx]: ugg. autoformatter here.  big oof. fixed in 3ea0cf313aac82ff12e6e728942121617efdf7da

Comment (anthonyshew) [reply docs/repo-docs/guides/tools/typescript.mdx]: Yeah, no worries. I only know because it bit me the other day. 😅

**Review by anthonyshew — APPROVED** [review]

**Pull request merged**